s3/pkg/datastore/azure/azure.go

```diff
@@ -331,6 +332,76 @@ func (ad *AzureAdapter) ListParts(ctx context.Context, multipartUpload *pb.ListP
 	return nil, ErrNotImplemented
 }
 
+func (ad *AzureAdapter) BackendCheck(ctx context.Context, backendDetail *pb.BackendDetailS3) error {
+
+	driver.RegisterDriverFactory(backendDetail.Type, &AzureBlobDriverFactory{})
+	backendDetailtry := &backendpb.BackendDetail{}
+	backendDetailtry.Id = backendDetail.Id
+	backendDetailtry.Name = backendDetail.Name
+	backendDetailtry.Type = backendDetail.Type
+	backendDetailtry.Region = backendDetail.Region
+	backendDetailtry.Endpoint = backendDetail.Endpoint
+	backendDetailtry.BucketName = backendDetail.BucketName
+	backendDetailtry.Access = backendDetail.Access
+	backendDetailtry.Security = backendDetail.Security
+
+	adap, err := driver.CreateStorageDriver(backendDetail.Type, backendDetailtry)
+	if err != nil {
+		log.Debug("failed to create storage. err:", err)
+		return err
+	}
+
+	endpoint := backendDetail.Endpoint
+	AccessKeyID := backendDetail.Access
+	AccessKeySecret := backendDetail.Security
+
+	credential, err := azblob.NewSharedKeyCredential(AccessKeyID, AccessKeySecret)
+
+	if err != nil {
+		log.Debug("create credential[Azure Blob] failed, err:%v\n", err)
+		return err
+	}
+
+	//create containerURL
+	p := azblob.NewPipeline(credential, azblob.PipelineOptions{
+		Retry: azblob.RetryOptions{
+			TryTimeout: MaxTimeForSingleHttpRequest,
+		},
+	})
+	URL, _ := url.Parse(endpoint)
+
+	containerURL := azblob.NewContainerURL(*URL, p)
+
+	object := pb.Object{
+		BucketName: backendDetail.BucketName,
+		ObjectKey:  "emptyContainer/",
+	}
+	objectId := object.BucketName + "/" + object.ObjectKey
+	blobURL := containerURL.NewBlockBlobURL(objectId)
+	bs := []byte{0}
+	stream := bytes.NewReader(bs)
+	options := azblob.UploadStreamToBlockBlobOptions{BufferSize: 2 * 1024 * 1024, MaxBuffers: 2}
```

Comment:
@vineela1999 please add constants here, remove the magic numbers

Reply:
code is removed